crates/nu-cli/src/commands/command.rs

```diff
@@ -238,6 +238,7 @@ pub trait WholeStreamCommand: Send + Sync {
 
 // Custom commands are blocks, so we can use the information in the block to also
 // implement a WholeStreamCommand
+#[allow(clippy::suspicious_else_formatting)]
```

Comment:
This was a weird one.  After updating to Rust 1.49, Clippy was throwing this warning:

```shell
warning: this looks like an `else if` but the `else` is missing
   --> crates/nu-cli/src/commands/command.rs:241:1
    |
241 | #[async_trait]
    | ^^^^^^^^^^^^^^
    |
    = note: `#[warn(clippy::suspicious_else_formatting)]` on by default
    = note: to remove this lint, add the missing `else` or add a new line before the second `if`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#suspicious_else_formatting
```

I assume this is a bug on their end, so I just temporarily turned the lint off.  Maybe I'm misinterpreting this?

Reply:
Yeah seems fine to disable that one. The crate we use might need to also update, but I'm fine ignoring the lint for now.